src/js/angular/sparql-editor/controllers.js

```diff
@@ -391,6 +391,8 @@ function SparqlEditorCtrl($scope, $location, $jwtAuth, $repositories, toastr, $t
             toastr.warning(notificationMessageEvent.message);
         } else if (NotificationMessageType.ERROR === notificationMessageEvent.messageType) {
             toastr.error(notificationMessageEvent.message);
+        } else if (NotificationMessageType.INFO === notificationMessageEvent.messageType) {
```

Comment:
This started moving aout of control. Can't we just do the following and omit this ugly if-else construct entirely? 
```javascript
toastr[notificationMessageEvent.messageType](notificationMessageEvent.message);
```

Reply:
This was last notification type, but I refactored it how you suggested.